src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -6495,5 +6495,24 @@ End Class
             VerifyItemIsAbsent(text, "X")
         End Sub
 
+        <WorkItem(4900, "https://github.com/dotnet/roslyn/issues/4090")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub ColorColorInstanceSharedRegression()
```

Comment:
Overall, it seems like a few more tests in this area would be good.  Did you look around for other existing tests?